pkg/applicationserver/applicationserver_util_test.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 var (
-	Timeout = (1 << 9) * test.Delay
+	Timeout          = (1 << 8) * test.Delay
```

Comment:
Note that this is still 4 seconds with `TEST_SLOWDOWN=16`

Can we get this lower and add `Timeout * X` in particular places that are notoriously slow?

Reply:
Travis runs on `TEST_SLOWDOWN=8`. With that being said, I can try to squeeze it to `(1 << 7)` and double it on a case-by-case basis.